luatest/hooks.lua

```diff
@@ -4,6 +4,30 @@ local comparator = require('luatest.comparator')
 
 local export = {}
 
+local preloaded_hooks = {
+    before_suite = {},
+    after_suite = {},
+
+    before_all = {},
+    after_all = {},
+}
```

Comment:
How about before/after each hooks? It seems, we need them in the `cluster` helper (#368).

Reply:
Added.

Reply:
I would add a test. I propose to add these hooks into `test_predefined_hooks`.

Reply:
Yep, added test for the `each` scope.